azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/arm/ResourceUtils.java

```diff
@@ -152,27 +152,30 @@ public static String nameFromResourceId(String id) {
      * @return the default api version to use
      */
     public static String defaultApiVersion(String id, Provider provider) {
-        String resourceType = resourceTypeFromResourceId(id).toLowerCase();
+        String resourceType = resourceTypeForApiVersion(id);
```

Comment:
Azure has a habit to return resource ids with types in uppercase, so it would be beneficial to keep that
```suggestion
        String resourceType = resourceTypeForApiVersion(id).toLowerCase();
```

Reply:
It's using `equalsIgnoreCase` later in the comparing loop so I just remove this `toLowerCase`.

Reply:
my bad - I didn't look close enough